db/migrate/20230510150542_add_user_ref_to_comments.rb

```diff
@@ -0,0 +1,5 @@
+class AddUserRefToComments < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :comments, :user, null: false, foreign_key: true
+  end
+end
```

Comment:
- [ ] It would be nice if you refer to author_id instead of the user_id because in the diagram you can see that there is an author which is an alias for the user so it would be nice if you follow the requirements and refer to the authro_id instead of user_id as you can see below ⬇️

```ruby
class AddUserRefToComments < ActiveRecord::Migration[7.0]
  def change
    add_reference :comments, :author, foreign_key: { to_table: 'users' }
  end
end
```